src/Tizen.NUI/src/public/Layouting/FlexLayout.cs

```diff
@@ -702,25 +710,40 @@ protected override void OnMeasure(MeasureSpecification widthMeasureSpec, Measure
         /// <param name="right">Right position, relative to parent.</param>
         /// <param name="bottom">Bottom position, relative to parent.</param>
         /// <since_tizen> 6 </since_tizen>
-        protected override void OnLayout(bool changed, LayoutLength left, LayoutLength top, LayoutLength right, LayoutLength bottom)
+        protected override void OnLayout (bool changed, LayoutLength left, LayoutLength top, LayoutLength right, LayoutLength bottom)
         {
 
             bool isLayoutRtl = Owner.LayoutDirection == ViewLayoutDirectionType.RTL;
             LayoutLength width = right - left;
             LayoutLength height = bottom - top;
 
             // Call to FlexLayout implementation to calculate layout values for later retrieval.
-            Interop.FlexLayout.FlexLayout_CalculateLayout(swigCPtr, width.AsDecimal(), height.AsDecimal(), isLayoutRtl);
+            Interop.FlexLayout.FlexLayout_CalculateLayout (swigCPtr, width.AsDecimal (), height.AsDecimal (), isLayoutRtl);
 
             int count = LayoutChildren.Count;
             for (int childIndex = 0; childIndex < count; childIndex++)
             {
                 LayoutItem childLayout = LayoutChildren[childIndex];
                 if (childLayout != null)
                 {
-                    // Get the frame for the child, start, top, end, bottom.
-                    Vector4 frame = new Vector4(Interop.FlexLayout.FlexLayout_GetNodeFrame(swigCPtr, childIndex), true);
-                    childLayout.Layout(new LayoutLength(frame.X), new LayoutLength(frame.Y), new LayoutLength(frame.Z), new LayoutLength(frame.W));
+                    if (!childLayout.Owner.TransformedByLayout)
```

Comment:
What do you think of using `SetFlexPositionType`?(https://yogalayout.com/docs/absolute-relative-layout)
Since the child with TransformedByLayout(false) is already added in children of yoga-layout at `InsertChild()`, i'm not sure that FlexLayout_CalculateLayout does work properly.

Reply:
Thanks, I used it and tested. Well works.